tests/test_trailedSourceFilter.py

```diff
@@ -94,15 +110,43 @@ def test_run_no_trails(self):
         np.testing.assert_array_equal(results.diaSources["diaSourceId"].values, [0, 1, 2, 3, 4])
         np.testing.assert_array_equal(results.trailedDiaSources["diaSourceId"].values, [])
 
+    def test_run_edge(self):
+        """Run trailedSourceFilterTask with the default max distance.
+        filtered out of the final results and put into results.trailedSources.
+        """
+        trailedSourceFilterTask = TrailedSourceFilterTask()
+
+        results = trailedSourceFilterTask.run(self.edgeDiaSources, self.exposure_time)
+
+        self.assertEqual(len(results.diaSources), 3)
+        np.testing.assert_array_equal(results.diaSources['diaSourceId'].values, [0, 2, 3])
+        np.testing.assert_array_equal(results.trailedDiaSources['diaSourceId'].values, [1, 4])
+
     def test_check_dia_source_trail(self):
```

Comment:
Please consider a clearer test name

Reply:
Which test are you suggesting a clearer name for? `test_check_dia_source_trail` is checking the `_check_dia_source_trail` function in trailedSourceFilter. I can rename that if you think that needs a clearer name.

Reply:
Yeah, I meant the one on this line, `test_check_dia_source_trail` - it felt like "test" and "check" were weirdly doubled but I see now why that is the name you chose! Hmm. Maybe just change the description, e.g., "Test that the DiaSource trail checker is correctly identifying long trails" ?

I'm realizing a lot of my uncertainty about what is happening is connected to the use of terms "filter", "mask", "flag", "check", "sort" semi-interchangeably. The renaming here to `trailed_source_mask` helps!